README.md

```diff
@@ -9,108 +9,73 @@ Modern C++ wrapper for MySQL C API
 
 Note: This library currently doesn't support MySQL library version 8 and newer.
 
-# Installation
+## Installation
 
-## Requirements
- - C++14 compatible compiler (tested GCC>=4.9)
- - MySQL C API dev (libmysqlclient-dev)
+### Requirements
 
-## Bootstrap
-We use git submodules for our dependencies and git requires you to initialize them manually.
-```bash
-git submodule update --init --recursive
-```
+- C++14 compatible compiler (tested GCC>=4.9)
+- MySQL C API dev (libmysqlclient-dev)
 
-## Build
-This is header only library therefore no build step is required.
+### Additional requirements for testing
 
-## Test
-Tests require docker(>=1.5.0) for running mysql instances with testing data.
-```bash
-make -j32 test
-```
-You may (among other things) specify custom compiler and extra flags.
-```bash
-make -j32 test CXX=/opt/szn/bin/g++ CXXEF=-Werror LDEF=-Wl,-rpath=/opt/szn/lib/gcc/x86_64-linux-gnu/current
-```
-
-## Install
-```bash
-make -j32 DESTDIR=/usr/local/ install
-```
+- socat
+- Boost System [for TEST_EXTENDED only] (libboost-system-dev)
 
+### Bootstrap
 
-# Packages
-We support several packages to be build by default:
- - Debian Stretch
- - Debian Jessie
- - Fedora 22
-
-## dbuild (docker-build)
-This can build packages in completely clean environment using docker.
-You might want to lower the `CONCURRENCY` in case you run out of memory.
-```bash
-make CONCURRENCY=32 package-debian-buster-dbuild
-```
-```bash
-make CONCURRENCY=32 package-debian-stretch-dbuild
-```
-```bash
-make CONCURRENCY=32 package-debian-jessie-dbuild
-```
-```bash
-make CONCURRENCY=32 package-fedora-22-dbuild
-```
+We use git submodules for our dependencies and git requires you to initialize them manually.
```

Comment:
Might worth noting, they're used only for testing afaik

Reply:
Good idea!